### DISCUSSION TITLE
Welcome and introductions

### DISCUSSION THREAD

**Discussion by sarahollis** [WorldHealthOrganization/godata#93] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by smbuthia** [comment]
🙋🏽‍♂️ I am Sam. Really exciting to have this space to share and engage! I'm an advocate for open source solutions such as Go.Data and I'm happy to support this work in any way I can.
